src/libse/Translate/Service/GoogleTranslator1.cs

```diff
@@ -63,7 +63,8 @@ public List<string> Translate(string sourceLanguage, string targetLanguage, List
                     input.Append(" " + SplitChar + " ");
                 }
 
-                var text = f.SetTagsAndReturnTrimmed(TranslationHelper.PreTranslate(p.Text.Replace(SplitChar.ToString(), string.Empty), sourceLanguage), sourceLanguage);
```

Comment:
This line is buggy `p.Text.Replace(SplitChar.ToString(), string.Empty)` will remove the \\n from \\r\\n which will make the line buggy!